src/app/shared/components/node-browser/node-browser-list/node-browser-list.component.scss

```diff
@@ -14,6 +14,56 @@ gtx-contents-list-item {
   cursor: pointer;
 }
 
+.icon-checkbox-wrapper gtx-checkbox {
+  position: absolute;
+  opacity: 0;
+  margin-left: 0.2%;
+  margin-top: 0.3%;
```

Comment:
It was the problem because `position ` property was absolute. For now works fine using  `item-primary ` class for displayName and giving some width to schemaName.